index.html

```diff
@@ -2,9 +2,178 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles/styles.css">
     <title>Title</title>
 </head>
 <body>
 
+
+<!---------------- HEADER ---------------->
+<header class="outer-container header-section">
+    <div class="inner-container">
+        <nav class="header-navigation">
+            <ul class="nav-list">
+                <li><a href="/">Case Studies</a></li>
+                <li><a href="/">Services</a></li>
+                <li><a href="/">Contact Us</a></li>
+            </ul>
+        </nav>
+
+        <div class="header-welcome-text">
+            <h1>Your website is an investment in the succes of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <button type="button">LET'S WORK TOGETHER</button>
+        </div>
+        <span class="header-img">
+                <img src="assets/squiggle_line_stripe.svg" alt="squiggly line stripe">
+        </span>
+
+    </div>
+</header>
+
+<!---------------- MAIN ---------------->
+<main class="outer-container">
+    <div class="inner-container">
+
+
+        <!--Onderstaande DIV is voor de introtekst onder de header-->
+        <div class="main-intro-container">
+            <p>Rekupe is a digital agency in Los Angeles</p>
+            <p>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</p>
+            <p>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</p>
+            <p class="main-intro-container-contact-us">Have a question? <a href="/">Click here</a> to contact us!</p>
+        </div>
+
+        <!--Onderstaande DIV is voor de articles en images 1 -->
+        <section class="main-sections">
+            <article class="main-article-text">
+                <h3>Rolling Stone</h3>
+                <div>
+                    <img src="assets/squiggle_line.svg" alt="squiggle_line"/>
+                </div>
+                <p>You know Rolling Stone. It's the definitive voice in music, politics and culture. It's a global
+                    icon.</p>
+                <p><a href="/">VIEW PROJECT</a></p>
+            </article>
+
+            <span class="article-image-main">
+                <img class="article-img" src="/assets/work-image-01.jpeg" alt="RollingStone laptop and magazine"/>
```

Comment:
```suggestion
                <img class="article-img" src="./assets/work-image-01.jpeg" alt="RollingStone laptop and magazine"/>
```
Anders kan ik de afbeelding niet zien (op mijn MacBook met Safari in ieder geval 😉 )